scanner/src/main/kotlin/storages/ProvenanceBasedFileStorage.kt

```diff
@@ -87,9 +87,11 @@ class ProvenanceBasedFileStorage(private val backend: FileStorage) : ProvenanceB
         val existingScanResults = read(provenance)
 
         if (existingScanResults.any { it.matches(scanResult) }) {
-            throw ScanStorageException(
+            logger.debug {
                 "Storage already contains a result for ${scanResult.provenance} and ${scanResult.scanner}."
-            )
+            }
```

Comment:
Without the wrapping exception a bit more context would be good, for example:
"Not writing scan result because storage..."
(same in the other stoarge class)